gcp-bundle/LICENSE

```diff
@@ -201,403 +201,656 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
+
 --------------------------------------------------------------------------------
 
-This binary artifact contains:
+Group: com.fasterxml.jackson.core  Name: jackson-annotations  Version: 2.18.2
+Project URL (from manifest): https://github.com/FasterXML/jackson
+Project URL (from POM): https://github.com/FasterXML/jackson
+License (from POM): The Apache Software License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
 Group: com.fasterxml.jackson.core  Name: jackson-core  Version: 2.18.2
-Project URL: https://github.com/FasterXML/jackson-core
-License: Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
-License: The Apache Software License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+Project URL (from manifest): https://github.com/FasterXML/jackson-core
```

Comment:
did this change because of a newer plugin version that was used to generate LICENSE/NOTICE?